appveyor.yml

```diff
@@ -1,15 +1,16 @@
 version: '{build}'
-build_script:
-- cmd: >-
-    set RUBY_VERSION=200
 
-    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+environment:
+    PATH:           'C:\Ruby200\bin;%PATH%'
+    VERSION_SUFFIX: -netstd
 
-    gem install bundler
+install:
+    - bundle install

```

Comment:
I think should have `cmd:` in front of it. Perhaps that is the default, and that is why it works, but we've specified it explicitly elsewhere.